website/src/main/webapp/website_script.js

```diff
@@ -117,6 +127,18 @@ function redirectId(id) {
   window.location = redirect;
 }
 
+function redirectRating(up, id) {
+  var rating = 'rating=good';
```

Comment:
Optional: Directly 0/1 to servlet instead of converting it from word to binary than back to word and then back to an integer within the servlet